content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In this chapter, we will learn how to download and use models and use the model
+registry to trigger automated CICD model workflows.
+
+## Downloading models
+
+It is useful to download model artifacts for example for local testing or for
+use in CICD workflows. With models versioned by DVC this can be done easily by
+using the `dvc get` CLI command as follows
+
+```bash
+dvc get https://<path-to-my-git-forge-account>/example-get-started-experiments models/model.pth --rev v1.0.0
```

Comment:
Any reason you don't use `dvc artifacts get`? 

Also, if you haven't, go to your Studio profile and enable the feature flag for `Enable model download feature on MDP`. This should be enabled by default soon, so let's write with the assumption that this is the recommended workflow for downloading models.

Reply:
I'm a bit confused, that command doesn't seem to exist. Do you mean this? https://dvc.org/doc/api-reference/artifacts_show#dvcapiartifacts_show

Reply:
https://dvc.org/doc/command-reference/artifacts/get

You might also want to take a look at https://github.com/iterative/dvc.org/pull/4939